samples/RXPTest/src/Tests/ViewBasicTest.tsx

```diff
@@ -106,6 +110,59 @@ class BasicView extends RX.Component<RX.CommonProps, RX.Stateless> {
                           'I have differing border radii' }
                     </RX.Text>
                 </RX.View>
+
+                <RX.View 
```

Comment:
Should we note somewhere that the intention of this is to test `accessibilityLabel` and `importantForAccessibility` on `Text` components?

Reply:
Actually it's mostly about View, not about Text. I will add comment in code.

---
In reply to: [161926917](https://github.com/Microsoft/reactxp/pull/431#discussion_r161926917) [](ancestors = 161926917)

Reply:
RX.Text implementation is still in the works. Maybe the tests for RX.Text specifically should be added once the implementation is in place....

Reply:
great catch! I'm removing incidental RX.Text tests from here and replacing them with explicit View tests.

---
In reply to: [161929548](https://github.com/Microsoft/reactxp/pull/431#discussion_r161929548) [](ancestors = 161929548)